README.md

```diff
@@ -6,62 +12,59 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/1342f842a14f40cca856d6e81204f8ac)](https://app.codacy.com/gh/Sudokuru/Frontend/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Maintainability](https://api.codeclimate.com/v1/badges/4cd1d1027b7b2532c680/maintainability)](https://codeclimate.com/github/Sudokuru/Frontend/maintainability)
 
-# Start the frontend app
-
-Run `npm i` in the root folder  
-Run `npm run start`
+## Sudokuru is an open-source project focused on developing a world-class, cross-platform Sudoku app. We aim to provide a delightful user experience while also contributing to the community by building a collection of reusable software modules. These modules are designed to be free, well-documented, modern, and interoperable, allowing developers to easily incorporate them into their own Sudoku-related projects.
 
```

Comment:
Could add a "Supported Platforms" section

### Supported Platforms

####  :white_check_mark: Tier 1 Support
- Web

#### :shipit: Teir 2 Support
- Android
- Windows
- Linux

#### :running: Planned Support
- iOS
- MacOS

#### Legend

 :white_check_mark: Tier 1 Support - We have full end to end test coverage for all new and existing functionality. We do manual testing of new features before deployment. 

:shipit: Teir 2 Support - We have partial or zero end to end test coverage for new and existing functionality. We may not always do manual testing of new features before deployment. 

:running: Planned Support - We do not currently support this platform, but we plan on supporting in the future.